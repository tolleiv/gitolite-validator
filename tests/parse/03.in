@group1 = hans.maulwurf homer.simpson
@group2 = hans.maulwurf homer.simpson bart.simpson
