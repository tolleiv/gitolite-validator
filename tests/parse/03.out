root:
  group:
    name: group1
    members:
      hans.maulwurf
      homer.simpson
  group:
    name: group2
    members:
      hans.maulwurf
      homer.simpson
      bart.simpson
